pkg/acsengine/defaults.go

```diff
@@ -238,3 +247,27 @@ func certGenerationRequired(a *api.Properties) bool {
 		return false
 	}
 }
+
+// getFirstConsecutiveStaticIPAddress returns the first static IP address of the given subnet.
+func getFirstConsecutiveStaticIPAddress(subnetStr string) string {
+	_, subnet, err := net.ParseCIDR(subnetStr)
+	if err != nil {
+		return DefaultFirstConsecutiveKubernetesStaticIP
+	}
```

Comment:
On second thought, this is the "defaults" path, which is just supposed to set the defaults and therefore it is not supposed to fail. It fails only because acs-engine sets the defaults _after_ it parses the customer's input json file, so I was forced to put the input parsing here.

@anhowe Would you like me to update and return an error for user input in this set-default path, or do you think you guys are going to change the acs-engine to set the defaults first and then load the user input file?

I did implement the error case, either way is fine for me.